### ISSUE TITLE
About baseline model

### ISSUE THREAD

**Issue by U-Help** [tztztztztz/eqlv2#2]
I have great interest in your work, and I want to ask a question. Does the baseline model also use sigmoid loss function rather than softmax loss function?

**Comment by tztztztztz** [comment]
Thanks for your interest. In this repo, the baseline model is trained with softmax loss function.

**Comment by U-Help** [comment]
Thanks for your reply. In your paper, at this location, you refer baseline model to sigmoid loss function,
![image](https://user-images.githubusercontent.com/39671748/112437414-e360c680-8d81-11eb-93e6-ca484fd8dd98.png)
whereas at corresponding picture, it becomes softmax loss function. I'm a little confused.
![image](https://user-images.githubusercontent.com/39671748/112437880-684be000-8d82-11eb-981d-4c06763d3d0c.png)

**Comment by tztztztztz** [comment]
Thanks for pointing out this typo. In Figure 2, we use the baseline models with the sigmoid loss function.  
We have already corrected it in the camera-ready version, which will be updated to arxiv soon.

**Comment by ShirleyHe2020** [comment]
Hi Authors, can this work be combined into YOLO v5 ？
